restcomm/restcomm.telephony.api/src/main/java/org/restcomm/connect/telephony/api/util/CallControlHelper.java

```diff
@@ -77,23 +81,41 @@ static boolean permitted(final String authorization, final String method, DaoMan
      * @throws IOException
      */
     public static boolean checkAuthentication(SipServletRequest request, DaoManager storage) throws IOException {
+        return checkAuthentication(request, storage, "MD5");
```

Comment:
@abdulazizali77 why do we hardcode to MD5 instead of using client-algorithm from the settings ?